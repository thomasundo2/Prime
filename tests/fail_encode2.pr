int main()
{
    lint x;
    printl(encode(x, "3"));
}