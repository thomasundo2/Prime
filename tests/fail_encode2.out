Fatal error: exception Failure("illegal argument void expected lint in encode(x, "3")")
